test/quantization/test_quantize.py

```diff
@@ -1051,6 +1099,45 @@ def test_train_save_load_eval(self):
                 out = model(x)
                 self.assertEqual(ref, out)
 
+    @override_qengines
+    def test_forward_hooks_preserved(self):
```

Comment:
Can you also add a test for quantize_dynamic() API?

Reply:
yes, sure. done

Reply:
@raghuramank100 ping :)